ote/src/cljs/ote/theme/colors.cljs

```diff
@@ -24,7 +26,6 @@
 (def basic-red "#ff5a05")
 (def basic-yellow "#ffd000")
 (def basic-gray "#9696aa")
-(def basic-brown "#784b28")
```

Comment:
I assume basic-brown is deleted from here because "hiekka - ruskea - beige" is not a regularly named "no-prefix color - dark color - light color" color continuation?